app/models/host_aggregate.rb

```diff
@@ -2,6 +2,9 @@ class HostAggregate < ApplicationRecord
   include SupportsFeatureMixin
   include NewWithTypeStiMixin
   include Metric::CiMixin
+
+  # TODO: this model doesn't have an event_where_clause, doesn't have a *_id in event_streams but includes
+  # event mixin.  Track down if we need timeline events for this as they look to not be completely supported yet.
```

Comment:
@agrare Do you know if HostAggregates can have events associated with them?

Reply:
See also the same question below in other models.

Reply:
HostAggregates are an openstack thing, I only see vm, host (like InfraManager::Host), and availability_zone in the openstack event parser

Reply:
The only model which does have events that was touched in this PR is PhysicalSwitch

Reply:
Do you have any idea why event mixin was included in host aggregates? 

That's ultimately the problem.  I'm trying to fix a bug in an unrelated model and the inconsistency in these other models becomes readily apparent when I try to expand the tests to cover the other models that pull in the event mixin so I can cover the failure with EmsCluster subclasses.  These others don't behave like the other event mixin types.  I hope it's a deletion but will need to dig into that in a followup.

Reply:
> Do you have any idea why event mixin was included in host aggregates?

Honestly if I had to guess, they copied another model as the basis for the HostAggregate class.

The include went all the way back to the first commit in this model
```
commit 5be79d08113b52daa183453fdd709aba73cf7e62
Author: Scott Seago <sseago@redhat.com>
Date:   Mon Jun 27 11:24:24 2016 -0400

    Host aggregate model code
```

Reply:
Are we good with a TODO to kick the can down the road?

Reply:
yeah, TODO is probably fine - no need to fix it all at once.  That being said, I still want to discuss because I still don't understand the overall issue nor the fix.

Reply:
I would think we'd unwind event_mixin inclusion in models that shouldn't need it.